content/docs/introduction/billing.md

```diff
@@ -0,0 +1,199 @@
+- Neon's _autoscaling_ feature, which allows you to set a minimum and maximum number of CUs for each compute endpoint. The number of active CUs scale up and down based on workload. _This feature is not yet available._
+- Neon's _always-on_ compute feature, which keeps one endpoint active indefinitely to avoid connection latency due compute endpoint startup time. _This feature is not yet available._
+
+The cost calculation for _Compute time_ is as follows:
+
+```text
+compute units * active time (hours) * price per hour
+```
+
+### Project storage
+
+The _Project storage_ metric counts the amount of data stored in all of your Neon projects. Project storage includes:
+
+- **The logical data size of your data**
+
+  This includes the size of all databases in your Neon projects, PostgreSQL SLRU (simple least-recently-used) caches, and a small amount of metadata. You can think of this as a _snapshot_ of your data at a point in time and similar to the data size that you would see in a standalone PostgreSQL installation.
+
+- **Retained Write-Ahead Log (WAL)**
+
+  The WAL is a record of data changes. Neon retains WAL to support _point-in-time restore_ and _database branches_.
+  - The _point-in-time-recovery window_ is _retained data history_ in the form of WAL records. The default point-in-time-restore window is seven days, which means that Neon stores seven days of data history. Data (WAL) that falls out of this window is evicted from storage and no longer counted toward project storage. The following diagram shows the primary branch of a Neon project (`main`) depicted as a timeline and a snapshot of your data that sits at the beginning of the point-in-time-restore window.
+
+      ```text
+      main   ---------########>
+                      ^        
+                  snapshot
+
+      Legend:
+
+      ####### point-in-time-restore window, which is 
+              retained data history in the form of WAL 
+              records
+
+      ------- data history (WAL) that has fallen out of the 
+              point-in-time-restore window, and can no
+              longer be accessed
+      ```
+
+  - A _database branch_ is a snapshot of your data (including the parent branch's retained history) at the point of branch creation combined with WAL records that capture data changes from that point forward.
+
+      When a branch is first created, it adds no storage. No data changes have been introduced yet and the branch's snapshot data still exists in the parent branch's point-in-time restore window, which means that it shares this data in common with the parent branch.
+
+      ```text
+      main   ---------########>
+                      ^      |
+                   snapshot  |
+                             |
+      branch A               #>
+                             ^
+                          snapshot  
```

Comment:
To me this illustration reads as if we create an extra snapshot on branch A, which we don't do.
Did you mean something different?

Reply:
I think I was confused by this phase "You can think of this as a snapshot of your data at a point in time and similar to the data size that you would see in a standalone PostgreSQL installation." ,a few lines above in logical size definition.

I read it as if snapshot means something material, not just a moment in time.
Not sure, how to word it better, so feel free to resolve this.